src/routes/(site)/s23positions/data.ts

```diff
@@ -0,0 +1,151 @@
+import { li } from '$lib/components/recursive-ul/utils';
+
+import type { ClubPosition } from './position';
+
+/**
+ * The tools that are used in the club.
+ *
+ * @remarks This is a map of tool names to their reasons for being used.
+ */
+export const TOOLS = {
```

Comment:
seem like a bit of a micro optimization that sacrifices readability for the sake of DRY. in fact id say this about the whole file, why not just use simple markdown strings for this?